lib/elixir/src/elixir_errors.erl

```diff
@@ -43,6 +45,10 @@ print_warning_group(Message, [Diagnostic | Others]) ->
   Locations = [["\n", n_spaces(Padding), "└─ ", 'Elixir.Exception':format_stacktrace_entry(ES)] || #{stacktrace := [ES]} <- Others],
   io:put_chars(standard_error, [Formatted, Locations, $\n, $\n]).
 
+get_span(#{span := nil}) -> nil;
+get_span(#{span := Span}) -> Span;
+get_span(_) -> nil.
```

Comment:
Can we remove this clause? Either the span is there or it isn't, no?

Reply:
Currently ~160 are failing if I remove this fallback clause, but indeed I think it makes sense for the key to always be there, since the diagnostic should be a map